### ISSUE TITLE
Android Auto should support browsing Library

### ISSUE THREAD

**Issue by hufman** [daneren2005/Subsonic#774]
It [looks like](https://github.com/daneren2005/Subsonic/blob/master/app/src/main/java/github/daneren2005/dsub/service/AutoMediaBrowserService.java#L118) Library browse support hasn't been implemented yet. This would be super cool to have at some point.

**Comment by hufman** [comment]
I implemented an initial attempt in https://github.com/hufman/Subsonic/tree/MediaBrowserLibrary
I tried testing it with the Android Auto Media Browser Simulator, and it seems to properly add browse entries to the menu. However, clicking on the Play/Shuffle/PlayLast entries doesn't seem to work on either my emulator or my phone. I don't have any personal Android Auto devices to test with either, but I might be able to find a friend to help test.

What are you using to test the Android Auto Media Browser functionality?
